src/blog/dark-mode-with-style-dictionary/index.mdx

```diff
@@ -0,0 +1,1352 @@
+
+Based on the example in the [Token structure](#Token-structure) section, we could have 2 files:
+
+```json
+// color/background.json
+{
+  "color": {
+    "background": {
+      "primary": { "value": "{color.core.neutral.0.value}" }
+    }
+  }
+}
+```
+
+```json
+// color-dark/background.json
+{
+  "color": {
+    "background": {
+      "primary": { "value": "{color.core.neutral.1000.value}" }
+    }
+  }
+}
+```
+
+Notice that the object structure of the token files is exactly the same for both light/default and dark mode, we are only overriding the value. Then we run Style Dictionary once with the only light/default token file and once with the dark token file to generate a set of light-mode outputs and dark-mode outputs.
+
+
+### Separate folders
+
+One way to set this up based on the multi-brand-multi-platform example is to have a folder that contains all the dark mode override tokens.
+
+<FileTree items={[{
+  name: 'tokens',
+  type: 'folder',
+  items: [{
+    name: 'color',
+    type: 'folder',
+    items: [{
+      type: 'json',
+      name: 'background.json'
+    },{
+      type: 'json',
+      name: 'core.json'
+    },{
+      type: 'json',
+      name: 'font.json'
+    },{
+      name: '...'
+    }]
+  }]
+},{
+  type: 'folder',
+  name: 'dark-tokens',
+  items: [{
+    type: 'json',
+    name: 'background.json'
+  },{
+    type: 'json',
+    name: 'font.json'
+  },{
+    name: '...'
+  }]
+}]} />
+
+Remember, the file and folder structure does not have any effect on the tokens themselves. The only difference is how you choose to organize your files. You could have a very different folder and file structure, this is just an example. 
+
+Then you would configure and run Style Dictionary twice with different configurations. To start you would have different source token files for light and dark mode:
+
+```javascript
+const StyleDictionary = require('style-dictionary');
+  
+// building default/light mode
+StyleDictionary.extend({
+  source: [
+    `tokens/**/*.json5`
+  ],
+  // ...
+}).buildAllPlatforms();
+  
+// building dark mode
+StyleDictionary.extend({
+  // Using the include array so that dark mode token overrides don't show warnings
+  include: [`tokens/**/*.json5`],
+  source: [`dark-tokens/**/*.json5`],
+  // ...
+}).buildAllPlatforms();
+```
+
+One difference with the [multi-brand][multi-brand] setup is here we don't want to loop through brands/themes and generate all the same files because we would be creating a lot of unnecessary code. Font sizes don't change in light/dark mode for example. That is why I think of these as color *modes* rather than *themes*. Instead looping through and running Style Dictionary twice with mostly the same configuration, here we run it once as default/light mode and then a second time only outputting necessary files needed to support dark mode.
```

Comment:
Instead looping through and running Style Dictionary twice with mostly the same configuration, here we run it once as default/light mode and then a second time only outputting necessary files needed to support dark mode. > Instead of looping through and running Style Dictionary twice with mostly the same configuration, we run it once as default/light mode and then a second time only outputting necessary files needed to support dark mode.